app/models/user_bookmark_list.rb

```diff
@@ -5,7 +5,7 @@ class UserBookmarkList
 
   PER_PAGE = 20
 
-  attr_reader :bookmarks, :per_page
+  attr_reader :bookmarks, :per_page, :posts, :topics
```

Comment:
It looks like we do not need this anymore?

Reply:
Ah yep will remove.